### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by hakman**

### CODE REVIEW

File: etcd-manager/images/BUILD [hunk 1]
```diff
@@ -92,10 +92,6 @@ container_image(
 
 container_image(
     name = "etcd-manager",
-    architecture = select({
-        "@io_bazel_rules_go//go/platform:amd64": "amd64",
-        "@io_bazel_rules_go//go/platform:arm64": "arm64",
-    }),
```
Comment (rifelpet) [inline etcd-manager/images/BUILD:98]: why are these being removed? have we tested that the resulting images are multi-arch?

perhaps we should follow the archived rules_docker suggestion and [move to rules_oci](https://github.com/bazelbuild/rules_docker?tab=readme-ov-file#status)

Comment (hakman) [reply etcd-manager/images/BUILD:98]: I am just experimenting. Bazel anything is huge pain. Tests are not yet passing due to build errors.
Not sure I want to move to rules_oci, just to strip it away in a few months.

Comment (hakman) [reply etcd-manager/images/BUILD]: Went back to the basics and it's good to go now.
@rifelpet Could you take another look?

Comment (hakman) [reply etcd-manager/images/BUILD]: BTW, for some reason this did not compile with Go 1.22...

**Review by rifelpet — APPROVED** [review]

**Pull request merged**